CRM/Utils/System/WordPress.php

```diff
@@ -293,7 +293,7 @@ public static function _showHTMLHead() {
    */
   public function mapConfigToSSL() {
     global $base_url;
-    $base_url = str_replace('http://', 'https://', $base_url);
+    $base_url = str_replace('http://', 'https://', $base_url ?? '');
```

Comment:
I don't know enough about wordpress for what it means for base_url to be null. In drupal I don't think it's possible, and in fact I thought base_url was a drupal thing. Was this just copy/paste from drupal and in fact does nothing?

Reply:
it does feel off but I was just going for the minimum approach

Reply:
Except then it hides the error forever. I think it's doing its job here and telling us this code is wrong. I couldn't find any reference to a global base_url in a wordpress install. Which would explain why it's null.